plugins/outputs/prometheus_client/README.md

```diff
@@ -40,6 +40,42 @@ This plugin starts a [Prometheus](https://prometheus.io/) Client, it exposes all
   ## enable mutually authenticated TLS connections
   # tls_allowed_cacerts = ["/etc/telegraf/clientca.pem"]
 
+  ## Use only tls ciphers defined in this list
```

Comment:
As the new options apply to many plugins, let's move this documentation here to a new document `docs/TLS.md` with a description of all the options in `internal/tls/config.go` and I'll link it in after we merge.

Reply:
I've added separate doc. Could you please review? I'm bad writer of docs:(

Reply:
@danielnelson , is it looks ok?